test/stores/notifications/RoomNotificationState-test.ts

```diff
@@ -92,6 +93,36 @@ describe("RoomNotificationState", () => {
         expect(listener).toHaveBeenCalled();
     });
 
+    it("updates on marked unread room account data", () => {
```

Comment:
"emits an `Update` event" is maybe a better description of what we are testing for here than "updates", but whatever